WooCommerce/Classes/ViewRelated/Dashboard/Factories/DashboardUIFactory.swift

```diff
@@ -19,11 +21,73 @@ protocol DashboardUI: UIViewController {
 }
 
 final class DashboardUIFactory {
-    static func dashboardUI() -> DashboardUI {
-        if FeatureFlag.stats.enabled {
-            return StoreStatsAndTopPerformersViewController(nibName: nil, bundle: nil)
+    private let siteID: Int
+    private let stores: StoresManager
```

Comment:
As mentioned in a previous comment, I am not sure about retaining the storesManager here.